PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -145,6 +157,7 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
   o2::framework::Configurable<float> centralityMin{"centralityMin", 0., "Minimum centrality"};
   o2::framework::Configurable<float> centralityMax{"centralityMax", 100., "Maximum centrality"};
   o2::framework::Configurable<bool> useSel8Trigger{"useSel8Trigger", true, "Apply the sel8 event selection"};
+  o2::framework::Configurable<bool> useUpcTrigger{"useUpcTrigger", true, "Apply the upc event selection"};
```

Comment:
```suggestion
  o2::framework::Configurable<bool> useUpcTrigger{"useUpcTrigger", false, "Apply the upc event selection"};
```
I'd not enable it by default

Reply:
Thanks for spoting this